README.md

```diff
@@ -50,11 +50,27 @@ You must set certain values in the `config.ini` file, in particular your Discor
     * `ServerID`: Your server ID.
     * `CommandPrefix`: What a message should begin with to be considered a command.
     * `BotName`: The name of your bot.
+* `[Emoji]`
     * `UpvoteEmoji`: The name of your upvote emoji (for the score function).
     * `DownvoteEmoji`: The name of your downvote emoji.
+    * `RedchikenEmoji`: The name of the redchiken emoji, used to vote on Banner of the Week Contests.
```

Comment:
while this is cute id prefer if the config was named semantically (banner vote emoji or something) - doesnt make much sense to have configuration say what the emoji is visually

Reply:
Resolved in a2c7d21